cmd/snap-seccomp/main_test.go

```diff
@@ -30,7 +30,7 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/seccomp/libseccomp-golang"
+	seccomp "github.com/seccomp/libseccomp-golang"
```

Comment:
Why was that one needed?

Reply:
Ah that was `goimports` ran by a plugin in my editor, not `gofmt`. I thought I had fixed it but must've reverted it. Thanks for spotting it, I'll fix it now